internal/pkg/aws/acm/acm.go

```diff
@@ -24,7 +30,46 @@ func New(s *session.Session) *ACM {
 	}
 }
 
-// ValidateCertAliases is a no-op and is supposed to validates if aliases are all valid against the provided ACM certificates.
+// ValidateCertAliases validates if aliases are all valid against the provided ACM certificates.
 func (a *ACM) ValidateCertAliases(aliases []string, certs []string) error {
+	validAliases := make(map[string]bool)
+	// TODO: Parallelize `findValidAliasesAgainstCert` if needed.
```

Comment:
1. Can we remove the TODO? I'm okay with writing it if it _must_ be done
2. Can we do this now?